src/main/java/sirius/biz/jdbc/tenants/ProfileController.java

```diff
@@ -67,8 +72,17 @@ public void profileChangePassword(WebContext ctx) {
                 userAccount.getLogin().verifyPassword(newPassword, confirmation, userAccount.getMinPasswordLength());
                 userAccount.getLogin().setCleartextPassword(newPassword);
                 oma.update(userAccount);
+
+                auditLog.neutral("%s changed her or his password.", UserContext.getCurrentUser().getUserName())
+                        .forCurrentUser()
+                        .log();
+
                 showSavedMessage();
             } catch (Exception e) {
+                auditLog.neutral("Ann error occurred when %s tried to changed her or his password: %s",
```

Comment:
Typo

Reply:
einfach ".. tried to change the password" ?